.github/workflows/R_lintr.yaml

```
@@ -29,5 +29,5 @@ jobs:
         shell: Rscript {0}
 
       - name: Lint
-        run: lintr::lint_package()
+        run: lintr::lint_package(linters = lintr::with_defaults(object_name_linter = NULL))
         shell: Rscript {0}
```

Why is this not 
```suggestion
        shell: Rscript -e {0}
```
?